recipes/rust/rust.inc

```diff
@@ -345,6 +338,8 @@ rust_do_install () {
 
         # Rust's install.sh doesn't mark executables as executable because
         # we're using a custom bindir, do it ourselves.
+	mkdir -p ${D}/${bindir}/
+	find ${D}/${prefix}/bin -type f -exec mv \{} ${D}/${bindir} \;
         chmod +x "${D}/${bindir}/rustc"
         chmod +x "${D}/${bindir}/rustdoc"
         chmod +x "${D}/${bindir}/rust-gdb"
```

Comment:
Since bindir isn't being used anymore, I'm pretty sure we can let rust's installer handle chmoding these